### PR TITLE
Bug 1911574: Expose volume mode on Upload Data form

### PR DESCRIPTION
**Pull request by glekner**
image

Exposed Volume mode and added the config map alert from the wizards.

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form.tsx [hunk 1]
```diff
@@ -108,25 +119,65 @@ export const UploadPVCForm: React.FC<UploadPVCFormProps> = ({
...
   const [requestSizeUnit, setRequestSizeUnit] = React.useState('Gi');
-  const [storageProvisioner, setStorageProvisioner] = React.useState('');
   const [isGolden, setIsGolden] = React.useState(!!osParam);
   const [os, setOs] = React.useState<OperatingSystemRecord>();
   const [osImageExists, setOsImageExists] = React.useState(false);
+  const defaultSCName = getDefaultStorageClass(storageClasses)?.metadata.name;
+  const updatedStorageClass = storageClasses?.find((sc) => sc.metadata.name === storageClassName);
+  const provisioner = updatedStorageClass?.provisioner || '';
+  let accessModes: string[] =
+    provisionerAccessModeMapping[provisioner] || getAccessModeForProvisioner(provisioner);
```
Comment (irosenzw) [inline frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form.tsx:145]: what happens when `provisioner` is `''` ?

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form.tsx:145]: if `provisioner` has a false value (e.g. '' or null) the `provisionerAccessModeMapping` map will return `undefined` and the `getAccessModeForProvisioner` method will return the default value `[rwo, rwx, rox]`

**Comment by yaacov** [comment]
/hold
need to re-run `yarn i18n`

**Pull request merged**